src/main/java/org/kainos/ea/api/BandService.java

```diff
@@ -10,7 +11,11 @@
 
 public class BandService {
 
-    private BandDao bandDao = new BandDao();
+    private BandDao bandDao;
+    public BandService(BandDao bd) {
+        bandDao = bd;
+    }
```

Comment:
Try not to abbreviate variable names down to letters, keep the constructor parameter as bandDao and then in the body use `this.bandDao = bandDao`. This will make it explicit for anyone reading the code later